src/Moryx.Products.UI/Connected Services/ProductService/Reference.cs

```diff
@@ -1255,6 +1270,73 @@ public Moryx.Products.UI.ProductService.Entry Properties
         }
     }
     
+    [System.Diagnostics.DebuggerStepThroughAttribute()]
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.Tools.ServiceModel.Svcutil", "2.0.2")]
+    [System.Runtime.Serialization.DataContractAttribute(Name="WorkplanModel", Namespace="http://schemas.datacontract.org/2004/07/Moryx.Products.Management.Modification")]
+    public partial class WorkplanModel : object
```

Comment:
The generator rearranged the order of generated models. Could you revert to lower part (manually)?